tasklist/els-schema/src/main/java/io/camunda/tasklist/util/ElasticsearchUtil.java

```diff
@@ -83,25 +83,34 @@ public static SearchRequest createSearchRequest(TemplateDescriptor template) {
 
   public static SearchHit getRawResponseWithTenantCheck(
       final String id,
-      IndexDescriptor descriptor,
-      QueryType queryType,
-      TenantAwareElasticsearchClient tenantAwareClient)
+      final IndexDescriptor descriptor,
+      final QueryType queryType,
+      final TenantAwareElasticsearchClient tenantAwareClient)
+      throws IOException {
+    return getRawResponseWithTenantCheck(
+        id, whereToSearch(descriptor, queryType), descriptor.getIndexName(), tenantAwareClient);
+  }
+
+  public static SearchHit getRawResponseWithTenantCheck(
```

Comment:
:wrench: We should bring back the `QueryType` when all indices have been rebased to the new `IndexDescriptor` abstract class, or remove it by the end of the refactoring. Same for the OpensearchUtil class.

Reply:
Yep, that's on my radar, once all have been moved, I will bring it back.